frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx

```diff
@@ -154,7 +154,7 @@ export const BackingStorage: React.FC<BackingStorageProps> = ({
   const supportedODFVendors = getSupportedVendors(odfCsv);
 
   const infraType = infra?.spec?.platformSpec?.type;
-  const enableRhcs = RHCS_SUPPORTED_INFRA.includes(infraType);
+  const enableRhcs = RHCS_SUPPORTED_INFRA.includes(infraType) && deployment === DeploymentType.FULL;
```

Comment:
this condition make sure , mcg standalone is not present RHCS but only IBM.
In 4.9 mcg standalone was blocked for any external storage